### DISCUSSION TITLE
Critique of Work

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#823] [Questions & Problems] [answered]
Good afternoon,

Is there anyone available to critique my recent PBPK modeling work as I prepare a manuscript for submission?

Paul

**Comment by StephanSchaller** [comment, marked as answer]
Hi Paul,

we can have a look if you want.
Let me know how you would like to exchange your work.

email is stephan.schaller@esqlabs.com

Best,
Stephan

**Comment by pburkat** [comment]
Great, I appreciate it. I will send to your email.
